channeld/channel_wire.csv

```diff
@@ -60,12 +60,10 @@ channel_init,,init_peer_pkt_len,u16
 channel_init,,init_peer_pkt,init_peer_pkt_len*u8
 channel_init,,reached_announce_depth,bool
 
-# Tx is deep enough, go!
+# master->channeld funding hit new depth >= lock depth
```

Comment:
Just to clarify, this message is sent from the time that lock depth is reached, until the time the announce depth is reached?

Reply:
Yes, though on reload it can actually fire for much greater depth in theory: stopping when we reach max(announce depth, lock depth) is an optimization.